src/Utils/Fallback.tsx

```diff
@@ -0,0 +1,34 @@
+export class Fallback {
+  private interval?: ReturnType<typeof setTimeout>;
+  private timeout?: ReturnType<typeof setTimeout>;
+
+  constructor(
+    private delayMs: number,
+    private pollingIntervalMs: number,
+    private pollingCb: () => Promise<boolean>
+  ) {}
+
+  async init() {
+    this.timeout = setTimeout(async () => {
```

Comment:
Add log here that fallback is initializing

Reply:
resolved

Reply:
`Fallback is initializing`

Reply:
Move it to the last line of the method

Reply:
resolved